docs/families.docs.js

```diff
@@ -0,0 +1,172 @@
+const familyEndpoints = {
+  '/families': {
+    post: {
+      tags: ['families'],
+      summary: 'Created object families in response',
+      description: 'families router',
+      operationId: 'postLoggedUser',
+      produces: ['application/json'],
+      parameters: [
+        {
+          in: 'header',
+          name: 'families',
+          description: 'Created object families',
+          required: true,
+          type: 'string',
+          schema: {
+            $ref: '#/definitions/FamiliesRequest',
+          },
+        },
+      ],
+      responses: {
+        '400': {
+          description: 'Request body format is invalid',
+        },
+        '401': {
+          description: 'User not authorized',
+        },
+        '409': {
+          description: 'User already created family/is a part of family',
+        },
+        '201': {
+          description: 'Family was created in DB successfully',
+          schema: {
+            $ref: '#/definitions/Families',
+          },
+        },
+      },
+    },
+    put: {
+      tags: ['families'],
+      summary: 'Update info from family document',
+      description: 'families router',
+      operationId: 'putLoggedUser',
+      produces: ['application/json'],
+      parameters: [
+        {
+          in: 'body',
+          name: 'Credentials',
+          description: 'User credentials',
+          required: true,
+          type: 'string',
+          schema: {
+            $ref: '#/definitions/Families',
+          },
+        },
+      ],
+      responses: {
+        '400': {
+          description: 'User request body format is invalid',
+        },
+        '401': {
+          description: 'User not authorized',
+        },
+        '409': {
+          description: 'User already created family / is a part of family',
+        },
+        '201': {
+          description: 'Family was created in DB successfully',
+          schema: {
+            $ref: '#/definitions/Families',
+          },
+        },
+      },
+    },
+  },
+
+  '/families/current': {
+    get: {
+      tags: ['families'],
+      summary: 'Get info from family document',
+      description: 'families router',
+      operationId: 'getLoggedUser',
+      produces: ['application/json'],
+      responses: {
+        '400': {
+          description: 'User request body format is invalid',
+        },
+        '401': {
+          description: 'User not authorized',
+        },
+        '409': {
+          description: 'User already created family / is a part of family',
```

Comment:
We're not returning 409 here. You could actually add 404 if family not found for that user